sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/unresolved.scala

```diff
@@ -645,4 +645,8 @@ case class TempResolvedColumn(child: Expression, nameParts: Seq[String]) extends
   override def dataType: DataType = child.dataType
   override protected def withNewChildInternal(newChild: Expression): Expression =
     copy(child = newChild)
+
+  override def sql: String = {
+    s"${child.sql}"
+  }
```

Comment:
This looks a little redundant. Shall we revise like the following?
```scala
override def sql: String = child.sql
```

Reply:
done!